### PR TITLE
Fix speech transform to handle macros correctly

### PR DESCRIPTION
**Pull request by CuriousVini**

### CODE REVIEW

File: src/main/java/io/cdap/plugin/gcp/speech/SpeechToTextTransform.java [hunk 1]
```diff
@@ -39,6 +39,7 @@
 import io.cdap.cdap.etl.api.TransformContext;
 import io.cdap.plugin.gcp.common.GCPConfig;
 import io.cdap.plugin.gcp.common.GCPUtils;
+import scala.annotation.meta.field;
```
Comment (albertshau) [inline src/main/java/io/cdap/plugin/gcp/speech/SpeechToTextTransform.java]: accidental?

File: src/main/java/io/cdap/plugin/gcp/speech/SpeechToTextTransform.java [hunk 2]
```diff
@@ -226,9 +228,9 @@ private void addTranscriptWithConfidence(List<StructuredRecord> speechArray,
   }
 
   private static class SpeechTransformConfig extends PluginConfig {
+    static final String NAME_TRANS_PART = "transcriptionPartsField";
+    static final String NAME_TRANS_TEXT = "transcriptionTextField";
```
Comment (albertshau) [inline src/main/java/io/cdap/plugin/gcp/speech/SpeechToTextTransform.java]: these can still be private right?

File: src/main/java/io/cdap/plugin/gcp/speech/SpeechToTextTransform.java (hunk 1 above)
Comment (CuriousVini) [reply src/main/java/io/cdap/plugin/gcp/speech/SpeechToTextTransform.java]: yea, fixed

File: src/main/java/io/cdap/plugin/gcp/speech/SpeechToTextTransform.java (hunk 2 above)
Comment (CuriousVini) [reply src/main/java/io/cdap/plugin/gcp/speech/SpeechToTextTransform.java]: right

**Review by albertshau — APPROVED** [review]

**Pull request merged**